tests/integration_tests/functional/test_api.py

```diff
@@ -472,10 +480,9 @@ def test_api_patch_pre_boot(test_microvm_with_api):
     assert test_microvm.api_session.is_status_bad_request(response.status_code)
     assert "Invalid request method" in response.text
 
-    # Partial updates to the machine configuration are not allowed.
+    # Partial updates to the machine configuration are allowed before boot.
     response = test_microvm.machine_cfg.patch(vcpu_count=4)
-    assert test_microvm.api_session.is_status_bad_request(response.status_code)
-    assert "Invalid request method" in response.text
+    assert test_microvm.api_session.is_status_no_content(response.status_code)
```

Comment:
Please verify with a `get` that the `PATCH` actually changes the object. Like [here](https://github.com/firecracker-microvm/firecracker/blob/73f94473467e017dc9f9bdb73c4e8db27b88484b/tests/integration_tests/functional/test_api.py#L96).

Reply:
Updated this one. Thanks for your review!